share/hooks/vcenter/create_vcenter_net.rb

```diff
@@ -158,8 +158,13 @@ def err_and_exit(error_message)
                 <controlPlaneMode>UNICAST_MODE</controlPlaneMode>\
                 <guestVlanAllowed>false</guestVlanAllowed>\
             </virtualWireCreateSpec>"
-        logical_switch = NSXDriver::VirtualWire
-                         .new(nsx_client, nil, tz_id, virtual_wire_spec)
+        begin
+            logical_switch = NSXDriver::VirtualWire
+                             .new(nsx_client, nil, tz_id, virtual_wire_spec)
+        rescue StandardError => e
+            STDERR.puts "ERROR: #{e.message}"
```

Comment:
This should also have configurable backtrace:

    if VCenterDriver::CONFIG[:debug_information]
        STDERR.puts "#{message} #{e.backtrace}"
    end

Reply:
Done